docs/reference/query-dsl/geo-distance-query.asciidoc

```diff
@@ -15,13 +40,18 @@ distance from a geo point. Assuming the following indexed json:
     }
 }
 --------------------------------------------------
+// CONSOLE
+// TEST[continued]
```

Comment:
Another option, so you don't have to do `// TEST[continued]` everywhere is to do `// TESTSETUP` but it only works with one snippet right now I think. So if you put the mapping and document into a single snippet you could use that instead. Then each snippet would have those commands run first. I think I'd prefer something like that because the snippets are examples rather than a single story? It is way too early with the annotations for me to have a good idea of what a good style is but that is my instinct at least.